_posts/2023-07-24-level-up-using-security-devices.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Level up your RubyGems Account Security with Security Devices
+layout: post
+author: Jenny Shen
+author_email: jenny.shen@shopify.com
+---
+
+In a [previous blog post](https://blog.rubygems.org/2023/04/11/security-device-cli-support.html), we introduced the latest updates to our security features, including CLI support for security devices and future plans for the feature.
+
+We are excited to announce that WebAuthn is fully integrated and is recognized to be the preferred choice for multi-factor authentication (MFA). When users register a security device, they will now be provided with recovery codes and be able to configure their [multi-factor authentication level](https://guides.rubygems.org/setting-up-multifactor-authentication/#authentication-levels) without needing to enable time-based one time password (OTP) MFA.
+
+### Why use a Security Device?
+Security devices offer an additional layer of protection that goes beyond passwords. By leveraging the [WebAuthn standard](https://www.yubico.com/authentication-standards/webauthn/), which utilizes public-key cryptography, they effectively mitigate the risk of password-related attacks. This includes phishing or credential stuffing.
+
+These devices not only provide enhanced security but also offer greater convenience compared to their time-based one time password counterparts, making them a preferred choice for authentication.
+
+While time-based codes from an authentication app can be intercepted or manipulated by phishing attacks, security devices eliminate the need for code inputs. They provide stronger protection against phishing attempts as they require the security key or the user (for biometric devices) to be present. This eliminates the risks associated with code-based authentication, ensuring a more robust defence against unauthorized attacks.
+
+### Why use a Security Device?
```

Comment:
How?

Reply:
Yep 🙈 copy paste mistake when I was copying over from a document. Thanks for catching it 🙏